<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>konfig_czynnosci</class>
 <widget class="QDialog" name="konfig_czynnosci">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>649</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Konfiguracja czynno≈õci</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1" rowspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_idc">
           <property name="text">
            <string>idc</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_idc"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_punkty">
           <property name="text">
            <string>punkty</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_punkty"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_opis">
           <property name="text">
            <string>opis</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEdit_opis"/>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBox_blokuj">
           <item>
            <property name="text">
             <string>NIE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>TAK</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>blokada</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_szukaj">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_szukaj">
           <property name="text">
            <string>szukaj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_szukaj">
           <item>
            <property name="text">
             <string>opis</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>idc</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>punkty</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_szukaj"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QTableView" name="tableView_czynnosci">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_przyciski">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_dodaj">
           <property name="text">
            <string>Dodaj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_edytuj">
           <property name="text">
            <string>Edytuj</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
     <zorder>tableView_czynnosci</zorder>
     <zorder>groupBox</zorder>
     <zorder>groupBox_szukaj</zorder>
     <zorder>verticalSpacer</zorder>
     <zorder>groupBox_przyciski</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
